probe/endpoint/nat.go

```diff
@@ -60,6 +60,10 @@ func (n natMapper) applyNAT(rpt report.Report, scope string) {
 		if !ok {
 			return
 		}
+		// Don't apply nat mappings to conntracked connections
+		if _, procspied := node.Latest.Lookup(Procspied); !procspied {
```

Comment:
Yes this was a old change (I thought this might have been one of the causes of the reversed connections).

We should do source natting in conntrack too, as this isn't going to work for most short lived connections (as this is periodic and only gets current connections).